src/FileSystemUtilities.js

```diff
@@ -51,10 +51,7 @@ export default class FileSystemUtilities {
 
   @logger.logifyAsync()
   static rimraf(filePath, callback) {
-    ChildProcessUtilities.exec("npm bin", {cwd: __dirname}, (err, npmBinPath) => {
-      if (err) return callback(err);
-      ChildProcessUtilities.spawn(join(npmBinPath.trim(), "rimraf"), [filePath], {}, callback);
-    });
+    ChildProcessUtilities.spawn(require.resolve("rimraf/bin"), [filePath], {}, callback);
```

Comment:
I'm using https://github.com/thlorenz/resolve-bin in multiple projects to get proper path to bin file.